.circleci/config.yml

```diff
@@ -283,8 +285,38 @@ jobs:
           name: Test with Firebase Test Lab
           command: >
             gcloud firebase test android run --type instrumentation \
-              --app integration-tests/build/outputs/apk/release/integration-tests-release.apk \
-              --test integration-tests/build/outputs/apk/androidTest/release/integration-tests-release-androidTest.apk \
+              --app integration-tests/build/outputs/apk/latestDependencies/release/integration-tests-latestDependencies-release.apk \
+              --test integration-tests/build/outputs/apk/androidTest/latestDependencies/release/integration-tests-latestDependencies-release-androidTest.apk \
+              --timeout 2m \
+              --results-bucket cloud-test-${GOOGLE_PROJECT_ID}
+      - run:
+          name: Copy test results data
+          command: |
+            mkdir -p ~/gsutil/
+            gsutil -m cp -r -U `gsutil ls gs://cloud-test-$GOOGLE_PROJECT_ID | tail -1` ~/gsutil/ | true
+          when: always
+      - store_artifacts:
+          path: ~/gsutil/
+      - store_test_results:
+          path: ~/gsutil/
+
+  run-firebase-tests-unityIAP:
```

Comment:
quick and dirty fix because I don't want this to block releasing a new version. I think if we update to the newer Android images in CircleCI we can skip running in firebase and possibly make this faster.